src/App.tsx

```diff
@@ -1,15 +1,35 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
```

Comment:
You don't need to import `React` from 'react', as you're not using `React.` something anywhere in your code. This can be removed since you’re using the JSX Transform that shipped with React 17.